src/test/xmeta/test_symbol_table.cpp

```diff
@@ -0,0 +1,92 @@
+#include "pch.h"
+
+#include <algorithm>
+#include <iostream>
+#include <string_view>
+
+#include "antlr4-runtime.h"
+#include "symbol_table.h"
+#include "XlangParser.h"
+#include "XlangLexer.h"
+#include "ast_to_st_listener.h"
+#include "helpers.h"
+
+using namespace antlr4;
+using namespace xlang::xmeta;
+
+xmeta_symbol_table st{ "" };
```

Comment:
Is this really required to be global? Prefer to simply initialize a new object for each test case.

Reply:
True, I'll fix that up.